index.html

```diff
@@ -146,48 +185,55 @@ <h1 class="about-me-title">About me</h1>
     </p>
     <button class="about-me-button">Get My Resume</button>
     <div class="about-me-line"></div>
-    <ul class="about-me-lists">
-      <li class="about-me-list-header">
-        <div class="star"></div>
-        <h1>Language</h1>
-      </li>
-      <li class="about-me-list-item">Javascript</li>
-      <li class="about-me-list-item">Ruby</li>
-      <li class="about-me-list-item">Html</li>
-      <li class="about-me-list-item">Css</li>
-    </ul>
-    <ul class="about-me-lists">
-      <li class="about-me-list-header">
-        <div class="square"></div>
-        <h1>Frameworks</h1>
-      </li>
-      <li class="about-me-list-item">Bootstrap</li>
-      <li class="about-me-list-item">Ruby on Rails</li>
-      <li class="about-me-list-item">RSpec</li>
-      <li class="about-me-list-item">CapyBara</li>
-      <li class="about-me-list-item">Selenium</li>
-    </ul>
-    <ul class="about-me-lists">
-      <li class="about-me-list-header">
-        <div class="circle"></div>
-        <h1>Skills</h1>
-      </li>
-      <li class="about-me-list-item">Codekit</li>
-      <li class="about-me-list-item">Github</li>
-      <li class="about-me-list-item">Codepen</li>
-      <li class="about-me-list-item">Gitlab</li>
-      <li class="about-me-list-item">Terminal</li>
-    </ul>
-
+    <div class="about-me-lists-container">
+      <ul class="about-me-lists">
+        <li class="about-me-list-header">
+          <div class="star"></div>
+          <h1>Language</h1>
+        </li>
+        <li class="about-me-list-item">Javascript</li>
+        <li class="about-me-list-item">Ruby</li>
+        <li class="about-me-list-item">Html</li>
+        <li class="about-me-list-item">Css</li>
```

Comment:
Please, modify your skills section to also match the design. The alignment of the design looks better. Also adjust the font sizes to match Figma.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187029719-92d51146-ddc4-453e-9c00-cefd04b44499.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/187029737-580d78e6-4fbb-48c7-83be-7639cb5a456a.png)